Arduino/MiP_ProMini_Pack_Library/src/mip.cpp

```diff
@@ -2183,6 +2459,18 @@ void MiP::processOobResponseData(uint8_t commandByte)
     case MIP_CMD_CLAP_RESPONSE:
         m_clapEvents.push(response[1]);
         break;
+    case MIP_CMD_GET_DETECTED_MIP:
+        m_detectedMiP.id = response[1];
```

Comment:
Will your CircularQueue class store a two element array (for those two-byte-only sends)?  I guess I'll give it a try later today.

Reply:
Implemented the above code fragment.  Declared irCode before switch statement because declaring it in `case MIP_CMD_GET_DETECTED_MIP:` caused a "crosses initialization of" error in Arduino IDE.